static/app/components/lastCommit.tsx

```diff
@@ -30,18 +31,26 @@ function LastCommit({commit, className}: Props) {
       return t('No message provided');
```

Comment:
in the case there is no commit message, maybe we should just display the link (if any) with the shortened commit ID as the text? i didn't implement this as i wanted to get opinions

Reply:
Yeah, I think this is a good idea. Should be the 1% or less case, but at least it's _some_ way to identify the commit.

Reply:
thanks!